modules/op-using-secrets-as-a-nonroot-user.adoc

```diff
@@ -15,4 +15,4 @@ In such scenarios, consider the following aspects of executing task runs and pip
 * SSH authentication for Git requires the user to have a valid home directory configured in the `/etc/passwd` directory. Specifying a UID that has no valid home directory results in authentication failure.
 * SSH authentication ignores the `$HOME` environment variable. So you must or symlink the appropriate secret files from the `$HOME` directory defined by {pipelines-shortname} (`/tekton/home`), to the non-root user's valid home directory.
 
-In addition, to configure SSH authentication in a non-root security context, refer to the link:https://github.com/tektoncd/pipeline/blob/main/examples/v1/taskruns/authenticating-git-commands.yaml[example for authenticating git commands].
+In addition, to configure SSH authentication in a non-root security context, refer to the `git-clone-and-check` step in the link:https://github.com/tektoncd/pipeline/blob/main/examples/v1/taskruns/authenticating-git-commands.yaml[example for authenticating git commands].
```

Comment:
I know that this already existed when you made this change, but per our [contribution guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites) we "cannot link to a repository that is hosted on www.github.com". "pipeline" is a repository for the tektoncd organization, right? Do you happen to know if this link was included by some special exception or not? If not, I feel like this should ideally be removed, as unfortunate as it would be to get rid of this seemingly useful example.

Reply:
I was under the impression that repositories maintained by Red Hat engineers were excepted, will await @adellape comment

Reply:
My understanding was https://github.com/redhat-developer was the only exception.

But yes, otherwise this reference should either be removed or ported and QE-verified to this repo in some way.

Reply:
I have requested Pipelines QE feedback on these examples.

Reply:
This one issue kept the PR dormant for two months. But it is now resolved, I have the new repo.